posthog/constants.py

```diff
@@ -300,3 +300,5 @@ class FlagRequestType(str, Enum):
 
 
 ENRICHED_DASHBOARD_INSIGHT_IDENTIFIER = "Feature Viewed"
+DATA_WAREHOUSE_TASK_QUEUE = "data-warehouse-task-queue"
+BATCH_EXPORTS_TASK_QUEUE = "no-sandbox-python-django"
```

Comment:
mental note: I should change the batch exports queue name.

Reply:
new mental note: Doing this would require being _really_ careful, so maybe I'll get over the fact we have a bad name for this queue :skull: